<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clients</class>
 <widget class="QDialog" name="Clients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>950</width>
        <height>588</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTableView" name="cltTabV"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Fiche client</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="cltLabelIdClient">
                <property name="text">
                 <string>IdClient</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cltCBoxCivilite">
                <item>
                 <property name="text">
                  <string>Mme</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mr</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="cltTxtNom">
                <property name="placeholderText">
                 <string>Nom</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="cltTxtPrenom">
                <property name="placeholderText">
                 <string>Prénom</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="cltTxtEmail">
                <property name="placeholderText">
                 <string>Adresse Email</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="cltTxtAdresse">
                <property name="placeholderText">
                 <string>Adresse</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="cltTxtVille">
                <property name="placeholderText">
                 <string>Ville</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="cltTxtCp">
                <property name="placeholderText">
                 <string>CP</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="cltTxtMob">
                <property name="placeholderText">
                 <string>Mobile</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="cltTxtTel">
                <property name="placeholderText">
                 <string>Téléphone</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="cltCBoxAbonne">
                <item>
                 <property name="text">
                  <string>Abonné(e)</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Non abonné(e)</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="cltBtnViderChamps">
                <property name="text">
                 <string>Vider les champs</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="cltBtnAjouter">
                  <property name="text">
                   <string>Ajouter</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="cltBtnModifier">
                  <property name="text">
                   <string>Modifier</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="cltBtnSupprimer">
                  <property name="text">
                   <string>Supprimer</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="cltBtnQuitter">
             <property name="text">
              <string>Quitter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cLabelAlerte">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cltCBoxCivilite</tabstop>
  <tabstop>cltTxtNom</tabstop>
  <tabstop>cltTxtPrenom</tabstop>
  <tabstop>cltTxtEmail</tabstop>
  <tabstop>cltTxtAdresse</tabstop>
  <tabstop>cltTxtCp</tabstop>
  <tabstop>cltTxtVille</tabstop>
  <tabstop>cltTxtTel</tabstop>
  <tabstop>cltTxtMob</tabstop>
  <tabstop>cltCBoxAbonne</tabstop>
  <tabstop>cltBtnViderChamps</tabstop>
  <tabstop>cltBtnAjouter</tabstop>
  <tabstop>cltBtnModifier</tabstop>
  <tabstop>cltBtnSupprimer</tabstop>
  <tabstop>cltBtnQuitter</tabstop>
  <tabstop>cltTabV</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
